src/pages/state/[state].js

```diff
@@ -216,18 +219,37 @@ export default function State({ data: { statesGeoData }, state = "DC" }) {
             state={stateData.abbreviation}
           />
           <EquitySection page="state" />
-          {stateData.videos?.length ? (
+          {stateData.videos?.length || stateData.voicesForReform?.length ? (
             <Row className={styles.voices}>
               <Row>
                 <h3>Voices For Reform</h3>
               </Row>
-              {stateData.videos.map(({ link, title, description }, i) => (
-                <Row key={`statewide-videos-${i}`}>
-                  <ReactPlayer url={link} width={594} height={361} />
-                  <h4>{title}</h4>
-                  {description && <ReactMarkdown>{description}</ReactMarkdown>}
+
+              {stateData.voicesForReform?.length ? (
+                <Row>
+                  <ul>
+                    {stateData.voicesForReform.map(n => (
+                      <li key={n.note + "vfr"}>
```

Comment:
Is `note` too much for a key? I use `i`s here pretty generously. Might otherwise give us issues, but we don't really have to worry about re-renders with our current setup anyway. Fine to leave it if it works

Reply:
I changed it to a string plus {i} 🤷